ParisBNF/abb/BNFabb161.xml

```diff
@@ -0,0 +1,370 @@
+        <fileDesc>
+            <titleStmt>
+              <title>Amharic commentary on Enoch, Vita of Fǝre Mikāʾel, Vita of Zarʾa ʾAbrǝhām, an Arabic treatise</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                      <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10088412q">BnF Éthiopien d'Abbadie 161</idno>
+                  </msIdentifier>
+                  
+                  <physDesc>
+                    <objectDesc form="Codex">
+                      <supportDesc>
+                        <support>
+                          <material key="paper"/>
+                        </support>
+                        <extent>
+                          <measure unit="page" quantity="362">2+358+2</measure>                             
+                          <dimensions type="outer" unit="mm">
+                            <height>230</height>
+                            <width>165</width>
+                          </dimensions>
+                        </extent>               
+                      
+                        <foliation>The manuscript is paginated. The first unpaginated leaf is assigned page numbers <locus target="#i #ii"/>.
+                          Only on <locus from="1r" to="2r">1-5</locus> and <locus target="#132">263-264</locus> 
+                          numbers are written on all pages, 
+                          in-between (the biggest part of the volume) the numbers are written only on recto-sides of the leaves. 
+                          The part accomodating <ref  target="#ms_i4"/> in Arabic is paginated in the opposite direction, from right to left,
+                          and the page numbers (from 1 to 94) are written only on the verso-sides of the leaves. The last leaf of the volume is not paginated 
```

Comment:
from 2 to 94?
Number 1 cannot possibly be written on a verso side, in continuous pagination recto is only for even numbers (2-4-6-8 etc)

Reply:
But it is on the verso side. Pagination of the Arabic part goes on the verso-sides, 1- 3-5-7... 93-(94 supplied by me, on the recto). The next number on the next verso is 264, and it switches to the recto and goes further again with odd numbers. It is quite confusing.

Reply:
No. For Arabic, just like for all others, the recto is the first page and verso is the second. The orientation is different. If you have 1-3-5-7 etc it is the pagination on recto. Then it is from 1 to 93.